DependencyInjection/Compiler/FormatListenerRulesPass.php

```diff
@@ -41,7 +41,7 @@ public function process(ContainerBuilder $container)
                 'path' => "^/$path/",
                 'priorities' => ['html', 'json'],
                 'fallback_format' => 'html',
-                'exception_fallback_format' => 'html',
+                'attributes' => array(),
```

Comment:
I think that's better to avoid having multiple format negotiator and instead allow to detect when the ExceptionController is used.
Our code is clearer and easier to maintain that way.